wordpress-mobile/WordPress-iOS
# ULS: Remove unifiedGoogle flag.

Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/477

This removes the `unifiedGoogle` feature flag. In Auth, the master `unifiedAuth` flag is used to show unified Google.

**Prerequisite:** You'll need a Google account that has WP 2FA setup via SMS.

To test:
1. Verify `Continue with WordPress` >` Continue with Google` shows the unified Google flow.
2. Select the Google account. Verify the 2FA view is the unified view.
3. On the 2FA view, go back and select the same Google account to trigger the social error. Verify error view does not show the option to signup.


PR submission checklist:
